api/GeolocationCoordinates.json

```diff
@@ -1,29 +1,36 @@
 {
   "api": {
-    "Coordinates": {
+    "GeolocationCoordinates": {
       "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Coordinates",
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/GeolocationCoordinates",
         "support": {
           "chrome": {
-            "version_added": "5"
+            "version_added": "5",
+            "notes": "Interface updated to <code>GeolocationCoordinates</code> in version 79."
```

Comment:
I think this is usually handled like this:

```
"chrome": [
  {
    "version_added": "79"
  },
  {
    "version_added": "5",
    "version_removed": "78",
    "alternative_name": "Coordinates"
  }
]
```

Reply:
Generally, if the browser supports a feature under a name not under the spec, then yes, that's how we handle it.  However, since `Coordinates` was a part of the spec, then maybe we should consider another option.  (Plus, the way this displays makes it seem like the API has only been around since recently, and doesn't suggest its true age.)

Maybe we should create a new subfeature called `Rename_to_GeolocationCoordinates` or something like that?

Reply:
I dunno, I think Joe's suggestion would probably be neater. Even though the alternative name field is usually used for non-standard names, is there anything to say that it can't be used for previously-standard names, as it were?

Reply:
There's not really anything that says alternative names HAVE to be for non-standard names and not previously-standard ones.  It's mostly just preference on how we want the data to be represented -- I have no preference either way!

Reply:
OK, I just wanted to make sure that it wouldn't be breaking any previously agreed convention. I'll make the change according to Joe's thoughts, and then let's have a look and see what we think.